src/components/code/code.tsx

```diff
@@ -21,27 +21,12 @@ import { CommonProps } from '../common';
 
 import React, { FunctionComponent, HTMLAttributes } from 'react';
 
-import { EuiCodeBlockImpl } from './_code_block';
+import { EuiCodeBlockImpl, EuiCodeBlockImplProps } from './_code_block';
 
-export interface EuiCodeSharedProps {
-  /**
-   * Sets the syntax highlighting for a specific language
-   * @see http://highlightjs.readthedocs.io/en/latest/css-classes-reference.html#language-names-and-aliases
-   * for options
-   */
-  language?: string;
-  transparentBackground?: boolean;
-}
+export type EuiCodeProps = CommonProps &
+  Omit<EuiCodeBlockImplProps, 'inline'> &
```

Comment:
```suggestion
  Pick<EuiCodeBlockImplProps, 'language' | 'transparentBackground'> &
```

or something similar that works in the props table (I didn't check). Only these two props are usable in `EuiCode`